src/imitation/scripts/common/demonstrations.py

```diff
@@ -91,9 +99,8 @@ def load_expert_trajs(
     Raises:
         ValueError: There are fewer trajectories than `n_expert_demos`.
     """
-    # wrap with str() in case rollout_path is a Path object
-    if "huggingface" in str(rollout_path):
-        rollout_path = expert.download_expert_rollouts(rollout_path=str(rollout_path))
+    if "huggingface" in rollout_type:
+        rollout_path = _download_expert_rollouts(rollout_type)
 
```

Comment:
Should we do input validation? Right now if I specify `rollout_type="huggingfce"` (typo) or `rollout_type="foobar"` it just defaults to disk.

Reply:
Added input validation. Changed the "disk" value of `rollout_type` to "local" since we can also generate expert trajectories by setting `rollout_path=None`. Let me know if it looks better or if we should change it.